builtin/providers/openstack/resource_openstack_fw_firewall_v2.go

```diff
@@ -0,0 +1,270 @@
+package openstack
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack/networking/v2/extensions/fwaas/firewalls"
+)
+
+func resourceFWFirewallV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceFirewallCreate,
```

Comment:
We'll want to add versions to these function names like in the other resource files.